easyenergy/easyenergy.py

```diff
@@ -20,6 +22,7 @@
 )
 from .models import Electricity, Gas
 
+API_HOST: str = "mijn.easyenergy.com"
```

Comment:
Please put this line in a separate `const.py` file

Reply:
Done